routes.js

```diff
@@ -1,17 +1,17 @@
-var Router = require('koa-router');
-var router = Router();
-var DB = require('./database');
-var db = DB();
+const Router = require('koa-router');
+let router = Router();
```

Comment:
Stylistic thing: require your libraries at the top of the file, then move everything else lower (like this). So I'd put this after the `const pug` line :)